bodhi/server/util.py

```diff
@@ -1473,3 +1474,19 @@ def _container_image_url(build, registry, tag=None):
         tag = '{}-{}'.format(build.nvr_version, build.nvr_release)
 
     return 'docker://{}/{}:{}'.format(registry, image_name, tag)
+
+def join_url(self, base, href):
+    """
+    Return a URL joined from base and href using library function 'urljoin' (lib: urlparse)
+    
+    For example, this can return a URL like
+    https://koji.fedoraproject.org/koji/search?terms=nginx-1.12-20180523111257.9c690d0e&type=build&match=glob
+
+    Args:
+        base (str): The base URL of the host.
+        href (str): The parameters component of the URL.
+
+    Returns:
+        str: A URL obtained by joining the two components.
+    """
+    return urlparse.urljoin(base, href)
```

Comment:
I thought we had talked about using ```urljoin()``` directly in the template? It doesn't make much sense to create a wrapper function around it.